app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -98,6 +102,11 @@ class PatientSearchResultItemView(
         assignedFacilityName = model.assignedFacilityName
     )
     renderIdentifier(model.identifier, searchQuery)
+    renderPatientStatus(model.status)
+  }
+
+  private fun renderPatientStatus(status: PatientStatus) {
+    patientDiedStatusView.visibleOrGone(isVisible = status == Dead)
```

Comment:
Looks like we didn't add a test for this. Is there a reason for that?

Reply:
This is a recycler view item layout. So, there aren't any tests for these 😅

Reply:
This is a recycler view item layout. There aren't tests for this particular item view. Also, unlike view renderer or UI actions we cannot verify that a particular UI call is made unless we provide another abstraction layer. Probably, something to think about in case tests is important in cases like this. 😅